guides/GITHUB.md

```diff
@@ -0,0 +1,343 @@
+contribute to. As you do this more, you'll learn and over time learn more
+about the actual code or documentation base. So in summary, don't
+be afraid to just jump in. You have meaningful questions to ask, or comments
+to make. You are the only one judging yourself for not
+having some perceived level of expertise.
+
+
+## What mindset should I have?
+
+Learning new things is challenging. It's important to realize that since using
+GitHub and git is entirely new to you, it's going to be challenging. You'll 
+likely start out and not entirely be sure about what the commands you are
+copy pasting are doing. There are a few things to remember here.
+
+### Inexperienced != Incapable
+
+Every current user of GitHub, or any technology for that matter, was new at 
+some point. Think of someone that you consider proficient or expert, and 
+realize that they at some point were in the same shoes as you. The
+fact that you are new to a technology says nothing about what you
+are capable of, of your innate abilities.
+
+### It's okay to ask for help
+
+What does it mean to ask for help? You might have questions, feedback, or
+a general comment, or just want to talk to someone. Asking for help can be daunting, 
+but guess what? The SourceCred community is open to it. Once you've
+sprinted over the mental hurdle that it's okay to ask, you'll very likely
+realize that engaging in the first place is a great way to learn, and quite a bit
+of fun. 
+
+
+### I will grow
+
+Some might say that you can only truly fail at something if you decide not to try.
+But now let's imagine that you try. Even if your progress feels slow, try mentally
+zooming ahead one year and think of the enormous amount you might learn. What's
+the trick to getting there? Just don't give up, and keep trying. At some point
+things that seem weird or challenging will just click. This is how learning
+tends to work.
+
+To summarize, if you have a growth mindset, and aren't afraid to ask for help,
+the experience can not only be successful in that you will learn how to use
+GitHub, you'll also have fun and connect with others. 
+
+
+## How do I contribute?
+
+Okay, let's get down to talking about GitHub! We could start by giving you high
+level concepts first, but intsead we are going to walk you through steps
+that you can follow along, and explain as we go.
+
+### 1. Create an account on GitHub
+
+This is the fun part! If you haven't yet, you get to [create an account](https://github.com/)
+which usually comes down to generating a username that you like and providing an email.
+This is exciting because when you think about it, GitHub is just another super
+geeky social network where you'll make a lot of new friends, and connect
+with friends that you already might have on their GitHub accounts.
+Once you have an account, don't forget to upload a profile picture and add
+any other metadata of interest under [settings](https://github.com/settings/profile).
+
+### 2. Install git
```

Comment:
I think we should guide folks to [GitHub for Desktop](https://desktop.github.com/) instead of using the command line. For people who are not exposed to using CLIs, they're very intimidating and a big learning curve that is outside of the scope of this gude.

Reply:
okay I modified it to remove the command line examples an direct folks either there (or more advanced) the command line getting started. Some new folks might be interested in at least reading how it goes on the command line.